src/main/java/net/dv8tion/jda/core/requests/restaction/PermOverrideData.java

```diff
@@ -0,0 +1,49 @@
+/*
+ *     Copyright 2015-2017 Austin Keener & Michael Ritter & Florian Spieß
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.dv8tion.jda.core.requests.restaction;
+
+import org.json.JSONObject;
+import org.json.JSONString;
+
+class PermOverrideData implements JSONString
+{
+    public static final int ROLE_TYPE = 0;
+    public static final int MEMBER_TYPE = 1;
```

Comment:
Why the magic numbers? Could this be an enum?

Reply:
Those are actual types used in the api, putting them in an enum doesn't make sense here since it's entirely internal and requires method calls to retrieve the int from the enum constant: `OverrideType.ROLE.getKey()`

Reply:
As a note, they aren't really magic numbers since they are specified by named constants.

Reply:
Fair enough